### ISSUE TITLE
Crash on fetching PHSL

### ISSUE THREAD

**Issue by goriunov** [dpguthrie/yahooquery#43]
**Describe the bug**

I am getting a crash when trying to pull premium financial information for PHSL as it is not returned as Data Frame, i am calling 
`tickers.p_all_financial_data(frequency="q")`  and data is returned as dict instead of dataframe, looks like it is unable to get some key. 

error: `'float' object has no attribute 'get'`

**To Reproduce**
1. Try to fetch premiun all financial data for PHSL

**Expected behavior**

Should return dataframe even if some keys are set to null

**Screenshots**

**Desktop (please complete the following information):**
 - OS: [e.g. iOS] POP OS linux
 - Browser [e.g. chrome, safari] chrome
 - Version [e.g. 22] 85

**Comment by dpguthrie** [comment]
There doesn't appear to be any data for this security.  If you go to link (https://finance.yahoo.com/quote/PHSL/financials?p=PHSL), you'll see that there's no data in the income statement, balance sheet, or cash flow tabs.  Also, when I try and retrieve `all_financial_data` here's what I'm getting:

```
>>> Ticker('PHSL').all_financial_data('q')
'Cash Flow data unavailable for PHSL'
```

**Comment by dpguthrie** [comment]
I see what's happening.  The first value returned is null and I'm expecting a dictionary, which is why you see the error above.  I'll apply the fix below in the next release (you can make the change on line 549 in ticker.py in the meantime if you'd like).

```
df["reportedValue"] = df["reportedValue"].apply(
    lambda x: x.get("raw") if isinstance(x, dict) else x
)
```

**Comment by goriunov** [comment]
Thank you for you work ! I will temporary change it on the fork but hope to see new version soon :)

**Comment by dpguthrie** [comment]
This should be fixed in 2.2.8.

```python
Ticker('PHSL').p_all_financial_data('q')

         asOfDate periodType  TotalRevenue
symbol
PHSL   2003-03-31         3M    12072000.0
PHSL   2003-06-30         3M     7738000.0
PHSL   2003-09-30         3M     8161000.0
```

**Comment by goriunov** [comment]
Thank you :)
